SPT-FOV-Fix-SIT/FovPatches.cs

```diff
@@ -115,7 +69,7 @@ public class OperationSetScopeModePatch : ModulePatch
         protected override MethodBase GetTargetMethod()
         {
             fAnimatorField = AccessTools.Field(typeof(FCSubClass), "firearmsAnimator_0");
-            weaponStateField = AccessTools.Field(typeof(FCSubClass), "gclass1605_0");
+            weaponStateField = AccessTools.Field(typeof(FCSubClass), "gclass1668_0");
```

Comment:
```suggestion
            weaponStateField = AccessTools.Field(typeof(FCSubClass), "gclass1678_0");
```

Are you sure it's not this one? I'm getting a Null Reference exception with 1668_0

Reply:
Great! Thank you for this one, as I'm not playing a lot with FOV Fix I missed this one!